gacl/docs/phpdoc/li_phpGACL.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
```

Comment:
did you mean to add these 3 files:
gacl/docs/phpdoc/li_phpGACL.html
gacl/docs/phpdoc/packages.html
gacl/docs/translations/russian/manual_rus.html

Reply:
guessing this is a merge  conflict issue (since I thin @tywrenn removed this stuff previously)

Reply:
Hi @bradymiller  sir

So should I remove those files?

Reply:
since they only exist now in this PR (ie. no longer in the main codebase), would remove themem unless they were purposely added back for a reason :)

Reply:
OK, remove it :)